case-lib/lib.sh

```diff
@@ -571,6 +575,34 @@ func_lib_check_sudo()
     }
 }
 
+func_lib_enable_pipewire()
+{
+    dlogi "Starting Pipewire..."
+    
+    sudo systemctl --global unmask pipewire{,-pulse}.{socket,service}
+    systemctl --user unmask pipewire{,-pulse}.{socket,service}
+    systemctl --user start pipewire{,-pulse}.{socket,service}
+    sudo systemctl --global unmask wireplumber.service 
+    systemctl --user unmask wireplumber.service
+    systemctl --user start wireplumber.service
+    systemctl --user daemon-reexec
```

Comment:
Please don't just "throw everything at it". It's called "programming by coincidence".

From `man systemctl`

> Reexecute the systemd manager. This will serialize the manager state, reexecute the process and deserialize the state again. This command is of little use except for debugging and package upgrades. Sometimes, it might be helpful as a heavy-weight daemon-reload.

Either `daemon-reload` is enough and you can drop this, or you know it's not enough for some documented reason (which you mention in a comment) and then you use only `daemon-reexec`. But not both.